nextflow.config

```diff
@@ -20,6 +21,14 @@ params {
     max_multiqc_email_size     = '25.MB'
     multiqc_methods_description = null
 
+// Fastp parameters
+    fastp_save_trimmed_fail                 = false
+    fastp_qualified_quality                 = 15
```

Comment:
opinion -- trimming this stringently will remove true biological diversity in the sample for low coverage organisms. I usually use 2 or 4. 2 is still a 50% chance it's accurate.

I think the only time i think of trimming more stringently is
1. variant calling
2. when I REALLY want to prove something isn't an artifact (like when i'm demonstrating my new method actually does what I say it does)

Reply:
Hmm ok, I will change the default behavior of this and add to the parameters help menu that this can be changed to more stringent